rns_shell/platform/graphics/x11/PlatformDisplayX11.cpp

```diff
@@ -39,6 +39,9 @@ std::unique_ptr<PlatformDisplay> PlatformDisplayX11::create(Display* display) {
 PlatformDisplayX11::PlatformDisplayX11(Display* display, bool displayOwned)
     : PlatformDisplay(displayOwned)
     , display_(display) {
+  auto screenDimension = screenSize();
```

Comment:
We have to avoid usage of auto,if we know the data type.
need to update with proper data type instead of auto

Reply:
sure